app/src/main/java/com/f0x1d/logfox/ui/viewholder/LogViewHolder.kt

```diff
@@ -18,6 +18,7 @@ import com.google.android.material.color.MaterialColors
 
 class LogViewHolder(
     binding: ItemLogBinding,
+    private val levelColorCacheMap: MutableMap<Int, Int>,
```

Comment:
I don't think viewHolder should receive `MutableMap`, at least just `Map`. Anyway maybe it's better to access it from adapter via `adapter()` method?